utils/_context/_scenarios/auto_injection.py

```diff
@@ -81,6 +83,8 @@ def __init__(
         include_almalinux_8_arm64=False,
         include_almalinux_9_amd64=False,
         include_almalinux_9_arm64=False,
+        include_redhat_79_amd64=False,
+        include_redhat_79_arm64=False,
```

Comment:
you also need to include in the constructor of InstallerAutoInjectionScenario (in this file)

Reply:
done.